core/src/test/java/com/graphhopper/routing/util/DataFlagEncoderTest.java

```diff
@@ -24,9 +27,11 @@
     private final EncodingManager encodingManager;
     private final int motorVehicleInt;
 
+    private final double DELTA = 0.1;
+
     public DataFlagEncoderTest() {
         encoder = new DataFlagEncoder();
-        encodingManager = new EncodingManager(encoder);
+        encodingManager = new EncodingManager(Arrays.asList(encoder), 8);
```

Comment:
Yes, this would be great. 

The real problem is that we are currently limited to 64 bit, where have an idea to make this kind of 'unbounded', but no time :/ ... see the [edge.getData(int) idea here](https://github.com/graphhopper/graphhopper/pull/798#issuecomment-248088079).